client/reader/sidebar/test/promo.jsx

```diff
@@ -2,9 +2,48 @@
  * @jest-environment jsdom
  */
 
-import { shallow } from 'enzyme';
+import editorReducer from 'calypso/state/editor/reducer';
+import mediaReducer from 'calypso/state/media/reducer';
+import siteSettingsReducer from 'calypso/state/site-settings/reducer';
+import timezonesReducer from 'calypso/state/timezones/reducer';
+import uiReducer from 'calypso/state/ui/reducer';
+import { renderWithProvider } from 'calypso/test-helpers/testing-library';
 import { ReaderSidebarPromo, shouldRenderAppPromo } from '../promo';
 
+const initialState = {
+	siteSettings: {},
+	sites: {
+		items: [],
+	},
+	media: {
+		queries: {},
+	},
+	currentUser: {
+		capabilities: {},
+	},
+	editor: {
+		imageEditor: {},
+	},
+	timezones: {
+		labels: {},
+		byContinents: {},
+	},
+	ui: {},
+};
+
+function renderWithRedux( ui ) {
+	return renderWithProvider( ui, {
+		initialState,
+		reducers: {
+			editor: editorReducer,
```

Comment:
I don't think you need any of these reducers here either. Looks like you just need `userSettings` as imported from `calypso/state/user-settings/reducer`.